OrbitGl/SessionsDataView.cpp

```diff
@@ -93,13 +104,33 @@ std::vector<std::string> SessionsDataView::GetContextMenu(
 void SessionsDataView::OnContextMenu(const std::string& a_Action,
                                      int a_MenuIndex,
                                      const std::vector<int>& a_ItemIndices) {
-  if (a_Action == MENU_ACTION_SESSIONS_LOAD) {
-    for (int index : a_ItemIndices) {
-      const std::shared_ptr<Session>& session = GetSession(index);
-      GOrbitApp->LoadSession(session);
+  if (a_Action == MENU_ACTION_LOAD) {
+    if (a_ItemIndices.size() != 1) {
```

Comment:
As mentioned above, this should never happen.  However, it would be great to be able to load multiple sessions in a single step.  This will require more than just the UI change.  Let's leave it as is for now until we find time to enable multi-sessions support.

Reply:
So loading multiple sessions (related to the same process) would mean loading the union of the modules and hooking the union of the functions, correct?

Reply:
Yes exactly.  It could be useful when profiling a game for example where we could have individual "animation", "physics" and "rendering" sessions, but one could be interested in 1, 2 or all 3 session presets at the same time.